src/app/components/publications/dialogs/add-publication-dialog.scss

```diff
@@ -0,0 +1,20 @@
+button:focus {
+  outline: none;
+}
+
+#addButton {
```

Comment:
Using https://mdbootstrap.com/docs/angular/components/buttons/ colors is probably better

Reply:
Mat icon buttons do not have the success color, but I just added it to the css